docs/Developer Resources/illumos Patches.rst

```diff
@@ -1,77 +1,64 @@
-OpenZFS Patches
+illumos Patches
 ===============
 
-The ZFS on Linux project is an adaptation of the upstream `OpenZFS
-repository <https://github.com/openzfs/openzfs/>`__ designed to work in
-a Linux environment. This upstream repository acts as a location where
-new features, bug fixes, and performance improvements from all the
-OpenZFS platforms can be integrated. Each platform is responsible for
-tracking the OpenZFS repository and merging the relevant improvements
-back in to their release.
-
-For the ZFS on Linux project this tracking is managed through an
-`OpenZFS tracking <http://build.zfsonlinux.org/openzfs-tracking.html>`__
-page. The page is updated regularly and shows a list of OpenZFS commits
-and their status in regard to the ZFS on Linux master branch.
-
-This page describes the process of applying outstanding OpenZFS commits
-to ZFS on Linux and submitting those changes for inclusion. As a
-developer this is a great way to familiarize yourself with ZFS on Linux
+The OpenZFS tracks illumos ZFS patches at the
+`illumos Tracking <http://build.zfsonlinux.org/openzfs-tracking.html>`__
+page. The page is updated regularly and shows a list of illumos commits
```

Comment:
It's not updated already, maybe something like this

```
The OpenZFS tracked illumos ZFS patches at the
illumos Tracking <http://build.zfsonlinux.org/openzfs-tracking.html>`__
page. Now the page isn't updated regularly and shows a list of illumos commits
and their status in regard to the OpenZFS master branch at the time of ZoL->OpenZFS transition.
```

Reply:
I wrote something similar.